lib/uk_postcode.rb

```diff
@@ -108,5 +108,3 @@ def digits(s)
     s && s.tr("IO", "10")
   end
 end
-
-require "uk_postcode/version"
```

Comment:
RubyMotion doesn't allow requiring in source files at runtime, Files all have to be included in the Rakefile at compile time.